README.md

```diff
@@ -1,2 +1,2 @@
 # buhari.js
-This package does nothing. Shouldn't Nigerian JavaScript developers make a package called Buhari? I think they should
+This package does nothing. Like Buhari.
```

Comment:
You probably don't want a full stop after the `nothing`, an ellipsis will do a better job. `does nothing...like Buhari`.

Reply:
Nah, a full stop is fine. Ellipses are overused.